shared/AppInsightsCore/src/JavaScriptSDK/ChannelController.ts

```diff
@@ -91,10 +91,19 @@ export class ChannelController implements ITelemetryPlugin {
                 arr[i - 1].setNextPlugin(arr[i]);
             }
             // Initialize each plugin
-            arr.forEach(queueItem => queueItem.initialize(config, core, extensions));
+            CoreUtils.arrForEach(arr, queueItem => queueItem.initialize(config, core, extensions));
 
             this.channelQueue.push(arr);
         }
     }
+
+    /**
+     * Static constructor, attempt to create accessors
+     */
+    // tslint:disable-next-line
```

Comment:
Disabling the tslint rules which don't like having a private, static, variables after declaring public members as we need for this to be the last definition as we NEED the ChannelController.prototype.getChannelControls has been defined.